evennia/objects/admin.py

```diff
@@ -95,7 +95,7 @@ class ObjectDBAdmin(admin.ModelAdmin):
     list_display = ("id", "db_key", "db_account", "db_typeclass_path")
     list_display_links = ("id", "db_key")
     ordering = ["db_account", "db_typeclass_path", "id"]
-    search_fields = ["^db_key", "db_typeclass_path"]
+    search_fields = ["^db_key", "db_typeclass_path", "id", "db_account__id"]
```

Comment:
- Since accounts and objects will have the same range of ids (starting from 1), filtering last on account will probably mean that account-id searching will rarely be relevant (since there are many more objects than accounts in a normal game). It'd say that it's probably better to filter on the account's `^db_key` field instead. 
- The filter order should also be changed, it makes more sense to prioritize `id` first by putting it first in the list.

Reply:
Ok. That makes sense. 
So Should we switch to exact search if a number is specifed.
That means if the user supplies a number we should show only exact match.
So id should be replaced by =id .

Reply:
Sure, sounds good, although I wonder if this is not automatically the case when querying with a primary key.

Reply:
I have updated my PR.